pkg/controller/podautoscaler/horizontal_test.go

```diff
@@ -466,8 +467,8 @@ func (tc *testCase) prepareTestClient(t *testing.T) (*fake.Clientset, *metricsfa
 			metrics := &cmapi.MetricValueList{}
 
 			// multiple objects
-			assert.Equal(t, "pods", getForAction.GetResource().Resource, "the type of object that we requested multiple metrics for should have been pods")
-			assert.Equal(t, "qps", getForAction.GetMetricName(), "the metric name requested should have been qps, as specified in the metric spec")
+			assert.Equal(t, "pods", getForAction.GetResource().Resource, fmt.Sprintf("got: %v, want: pods", getForAction.GetResource().Resource))
```

Comment:
no need for the extra `fmt.Sprintf`, IIRC -- the terminating arguments here are themselves a format string.

Reply:
also, what's with this change -- we already have a "want" and "got" as part of the assert.  The message is just providing more context, and the way you've changed it provides less.